README.md

```
@@ -0,0 +1,110 @@
+### Failed Check Logs
+
+```bash
+# Logs for failed checks only; no output if all pass
+gh-pr-context logs
+```
+
+## Output Format
+
+Comments are returned as terse plain-text with clear delimiters. Review comment replies are nested under their parent. Example:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)